test/listener_test.js

```diff
@@ -10,6 +10,10 @@ suite('TaskListener', () => {
 
     let cfg = config({profile: 'test'});
 
+    if (process.env.NODE_ENV === 'development') {
```

Comment:
I think the right condition here is that there are no pulse credentials configured.  We do want to run these tests if we have credentials..